cvmfs/shortstring.h

```diff
@@ -175,15 +176,34 @@ class ShortString {
     return ShortString(this->GetChars() + start_at, length-start_at);
   }
 
-  static uint64_t num_instances() { return atomic_read64(&num_instances_); }
-  static uint64_t num_overflows() { return atomic_read64(&num_overflows_); }
+  static void RegisterCounters(perf::Statistics *statistics) {
```

Comment:
Let's try like this: add a function `perf::TryInc` that will only increase the counter if it is not `NULL`.  This new function should be used instead of `perf::Inc` in the `ShortString` class (but only there).